### PR TITLE
Add fabric-peer docker image readme

### PR DESCRIPTION
**Pull request by denyeart**
#### Type of change

- Documentation update (for peer docker image)

#### Description
Add a README.md alongside the fabric-peer image Dockerfile.
The README.md will be used for the fabric-peer dockerhub information.
In the future, the README.md could be automatically deployed
to dockerhub when the fabric-peer image is pushed.

Signed-off-by: David Enyeart <enyeart@us.ibm.com>

### CODE REVIEW

File: images/peer/README.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
+# Quick reference
+
+-	**Where to get Docker help**:  
+	[the Docker Community Forums](https://forums.docker.com/), [the Docker Community Slack](http://dockr.ly/slack), or [Stack Overflow](https://stackoverflow.com/search?tab=newest&q=docker)
+
+-	**Where to get Hyperledger Fabric help**:  
+	[Fabric documentation](https://hyperledger-fabric.readthedocs.io/en/latest/), [Stackoverflow](https://stackoverflow.com/questions/tagged/hyperledger-fabric), or [Rocket.Chat](https://chat.hyperledger.org/channel/fabric)
```
Comment (ryjones) [inline images/peer/README.md]: `Chat`, not `Rocket.Chat`

File: images/peer/README.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+You can run a Fabric peer container as follows:
+
+```console
+$ docker run -d --publish 7051:7051 \
+-v /tmp/fabric/config/peer0.org1.example.com:/etc/hyperledger/fabric \
+-v /tmp/fabric/crypto-config/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/msp:/etc/hyperledger/fabric/msp \
+-v /tmp/fabric/crypto-config/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls:/etc/hyperledger/fabric/tls \
+-v /tmp/fabric/data/peer0.org1.example.com:/var/hyperledger/production \
+--name peer0.org1.example.com hyperledger/fabric-peer:2.0 peer node start
```
Comment (lindluni) [inline images/peer/README.md:51]: `peer node start` is the command the container is launched with via the Dockerfile, no need to pass it into the run command

File: images/peer/README.md (hunk 1 above)
Comment (ryjones) [reply images/peer/README.md]: maybe link here as well? https://wiki.hyperledger.org/display/CA/Logging+in+to+chat

File: images/peer/README.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+Inside this directory you'll find a default core.yaml configuration file with a SampleOrg organization (aka mspid) configured.
+
+You can provide your own `core.yaml` configuration file on your host, and then mount this directory inside your peer container at `/etc/hyperledger/fabric`, thereby overriding the default peer configuration. For example, your core.yaml may
+set `peer.localMspId` to your own organization's mspid.
+
+The `docker run` command above demonstrates how to mount such a configuration directory from your host to the
+peer container's `/etc/hyperledger/fabric` directory.
+
+Alternatively, you can override individual `core.yaml` configuration properties by passing
+environment variables to the peer container, for example CORE_PEER_LOCALMSPID.
```
Comment (MHBauer) [inline images/peer/README.md]: should CORE_PEER_LOCALMSPID be backtick verbatim? `CORE_PEER_LOCALMSPID`

File: images/peer/README.md (hunk 1 above)
Comment (denyeart) [reply images/peer/README.md]: added

File: images/peer/README.md (hunk 2 above)
Comment (denyeart) [reply images/peer/README.md:51]: understood, but i think it helps people understand what is going on, e.g. they may want to override `peer node start` with something like `peer node reset` to reset the peer's ledger, or they may want to use the binaries instead.

**Review by ryjones — APPROVED** [review]
lgtm

**Review by sykesm — APPROVED** [review]

**Pull request merged**